docs/getting-started-guides/kubeadm.md

```diff
@@ -277,53 +296,67 @@ See the [list of add-ons](/docs/admin/addons/) to explore other add-ons, includi
 
 * Slack Channel: [#sig-cluster-lifecycle](https://kubernetes.slack.com/messages/sig-cluster-lifecycle/)
 * Mailing List: [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
-* [GitHub Issues](https://github.com/kubernetes/kubernetes/issues): please tag `kubeadm` issues with `@kubernetes/sig-cluster-lifecycle`
+* [GitHub Issues in the kubeadm repository](https://github.com/kubernetes/kubeadm/issues)
 
 ## kubeadm is multi-platform
 
 kubeadm deb packages and binaries are built for amd64, arm and arm64, following the [multi-platform proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/multi-platform.md).
 
 deb-packages are released for ARM and ARM 64-bit, but not RPMs (yet, reach out if there's interest).
 
-ARM had some issues when making v1.4, see [#32517](https://github.com/kubernetes/kubernetes/pull/32517) [#33485](https://github.com/kubernetes/kubernetes/pull/33485), [#33117](https://github.com/kubernetes/kubernetes/pull/33117) and [#33376](https://github.com/kubernetes/kubernetes/pull/33376).
-
-However, thanks to the PRs above, `kube-apiserver` works on ARM from the `v1.4.1` release, so make sure you're at least using `v1.4.1` when running on ARM 32-bit
-
-The multiarch flannel daemonset can be installed this way.
+Currently, only the pod network flannel is working on multiple architectures. You can install it this way:
 
     # export ARCH=amd64
     # curl -sSL "https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml?raw=true" | sed "s/amd64/${ARCH}/g" | kubectl create -f -
 
 Replace `ARCH=amd64` with `ARCH=arm` or `ARCH=arm64` depending on the platform you're running on.
 Note that the Raspberry Pi 3 is in ARM 32-bit mode, so for RPi 3 you should set `ARCH` to `arm`, not `arm64`.
 
-## Limitations
+## Cloudprovider integrations (experimental)
 
-Please note: `kubeadm` is a work in progress and these limitations will be addressed in due course.
-Also you can take a look at the troubleshooting section in the [reference document](/docs/admin/kubeadm/#troubleshooting)
+We have got a lot of requests of how to enable a specific cloudprovider. This is possible, but currently requires manual configuration, 
+and therefore not supported. As a first step if you want to do it, edit the `kubeadm` dropin for the `kubelet` service
```

Comment:
Suggest: "Enabling specific cloud providers is a common request, this currently requires manual configuration and is therefore not yet supported. If you wish to do so, edit the..."